### PR TITLE
feat(interview-prep): Porting Rosetta problems

### PR DESCRIPTION
**Pull request by NitronR**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `dev` branch.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/challenge-tests`)
- [x] All new and existing tests pass the command `npm test`.
- [x] Use `npm run commit` to generate a conventional commit message.
    Learn more here: <https://conventionalcommits.org/#why-use-conventional-commits>
- [x] The changes were done locally on your machine and NOT GitHub web interface.
    If they were done on the web interface you have ensured that you are creating conventional commit messages.

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, so sorry for the delay! I left some suggestions for changes as a review.

File: challenges/08-coding-interview-prep/rosetta-code.json [hunk 1]
```diff
@@ -11596,6 +11596,844 @@
...
+          ]
+        }
+      }
+    },
+    {
+      "title": "Ordered words",
+      "description": [
+        "An  <i>ordered word</i>  is a word in which the letters appear in alphabetic order.",
+        "Examples include  <b>abbey</b>  and  <b>dirt</b>.",
+        "Write a function that returns true if given word is ordered. Otherwise returns false"
```
Comment (scissorsneedfoodtoo) [inline challenges/08-coding-interview-prep/rosetta-code.json]: "Write a function that returns true if the letters of the given word are in alphabetic order and false if they're not."

File: challenges/08-coding-interview-prep/rosetta-code.json [hunk 2]
```diff
@@ -11596,6 +11596,844 @@
...
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Pangram checker",
+      "description": [
+        "A pangram is a sentence that contains all the letters of the English alphabet at least once.",
+        "For example:  <i>The quick brown fox jumps over the lazy dog</i>.",
+        "Write a function to check if a sentence is a  <a href=\"https://en.wikipedia.org/wiki/Pangram\">pangram</a>."
```
Comment (scissorsneedfoodtoo) [inline challenges/08-coding-interview-prep/rosetta-code.json]: "Write a function to check if a sentence is a  <a href=\"https://en.wikipedia.org/wiki/Pangram\">pangram</a>. It should return true if the given sentence is a pangram, and false if it isn't."

**Comment by NitronR** [comment]
@scissorsneedfoodtoo Changes done. Thanks for the review and fixing the descriptions.

**Comment by scissorsneedfoodtoo** [comment]
Alright! Thank you as always for your contribution :+1:

**Pull request merged**